test_requirements.txt

```diff
@@ -1,7 +1,7 @@
 celery==3.1.18
 codecov
-coverage==4.4.2
-cryptography==1.9
+coverage
```

Comment:
why unpin this? this can cause issues in the future.

Reply:
pinning this back